README.md

```diff
@@ -140,7 +140,7 @@ The blue team's homezone is always on the left and the red team's homezone is al
 The full documentation for the `setup_test_game` function can be read in its docstring:
 ```bash
 $ python
->>> from utils import setup_test_game
+>>> from pelita.utils import setup_test_game
```

Comment:
Why did you remove the `utils.py` module? It was there for a reason, i.e. to show that you can have files with a collection of utility functions that you can then import in the modules for the move function. Unfortunately after the last refactoring we couldn't come up with any useful function to be added there, so we just had the two imports from pelita, but we want to have this module, and we want them to import from it and not from pelita directly. If you can come up with a useful utility function, please add it there.
But, IMHO, all the following changes where you did `from utils import`⟶`from pelita.utils import` should be reverted.

Reply:
Please see the commit message. It answers exactly this question.

Reply:
> message

OK, as I said, I agree with the gist of this, I am saying that we want to **have** a utils-like module to show how this works. I don't want it to go just now, because otherwise we will forget about it and won't add anything.

Reply:
I added a new module in `talk_util.py` with a `say_underlined` function.